scene/resources/canvas_item_material.cpp

```diff
@@ -159,6 +159,9 @@ void CanvasItemMaterial::flush_changes() {
 }
 
 void CanvasItemMaterial::_queue_shader_change() {
+	if (!is_initialized)
+		return;
```

Comment:
Style issue:
```suggestion
	if (!is_initialized) {
		return;
	}
```
(Same for 3D)